<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewShape</class>
 <widget class="QDialog" name="NewShape">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Shape</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QSpinBox" name="lSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>128</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>110</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="penStyleLabel">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>293</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Style:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="capStyleComboBox">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>291</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Flat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Square</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Round</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="brushStyleLabel">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>293</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Style:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="brushColorComboBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>251</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>White</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Magenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gray</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="brushStyleComboBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>291</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Solid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ver</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No Brush</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="sPointLabel">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Point:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="penColorComboBox">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>251</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>White</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Magenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gray</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="brushLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>213</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="penLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>213</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="yLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>112</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QComboBox" name="penStyleComboBox">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>291</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Solid Line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dash Line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dot Line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dash Dot Line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dash Dot Dot Line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No Pen</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="wSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>128</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="xSpinBox">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="brushColorLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>253</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QLabel" name="joinLabel">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>254</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Join Style:</string>
   </property>
  </widget>
  <widget class="QLabel" name="penColorLabel">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>253</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ySpinBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QComboBox" name="penJoinComboBox">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>251</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Miter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bevel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Round</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="wLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="capLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>292</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cap Style:</string>
   </property>
  </widget>
  <widget class="QLabel" name="shapeIDLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shape ID:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="shapeTypeComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>18</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Square</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ellipse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Circle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polyline</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polygon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Text</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="shapeTypeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shape Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel_2">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>140</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="sPointLabel_2">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>End Point:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yLabel_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>142</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="xEndSpinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="yEndSpinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>387</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>427</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel_2">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>430</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="penWidthSpinBox">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>323</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>325</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="textColorComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>458</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>White</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Magenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gray</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="textColorLabel">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>460</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pointSizeSpinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>525</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>525</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Size:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fontFamilyComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>427</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Comic Sans MS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Courier</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Helvetica</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Times</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>429</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Family:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fontStyleComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>458</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Italic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oblique</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Style:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fontWeightComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>488</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Light</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bold</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>488</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Weight:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dimensionsLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Dimensions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>588</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="shapeIDSpinBox">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>17</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>195</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>365</y>
     <width>590</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>553</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>20</width>
     <height>512</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>20</width>
     <height>512</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>575</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="alignmentComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>488</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Top</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bottom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Center</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>492</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alignment:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPointButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>494</x>
     <y>107</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearPointsButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>494</x>
     <y>138</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Points</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointsLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>175</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Points:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="idTakenLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pointCountLabel">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>177</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
